Makefile

```diff
@@ -29,6 +34,21 @@ tools: $(TOOLS)
 $(TOOLS): oci-%:
 	go build -ldflags "-X main.gitCommit=${COMMIT}" ./cmd/$@
 
+.PHONY: man install uninstall
+
+_man:
+	go-md2man -in "cmd/oci-unpack/oci-unpack.1.md" -out "oci-unpack.1"
+	go-md2man -in "cmd/oci-image-validate/oci-image-validate.1.md" -out "oci-image-validate.1"
+	go-md2man -in	"cmd/oci-create-runtime-bundle/oci-create-runtime-bundle.1.md" -out "oci-create-runtime-bundle.1"
+
+install: _man
```

Comment:
Phony targets as leaf-targets are fine.  But if you put them in a chain you'll always rebuild leafier targets.  So I'd like to see:

    .PHONY: … install man tools …

    man: $(MAN)
    tools: $(TOOLS)
    install: $(MAN) $(TOOLS)
      …

That way you don't re-install if none of the sources changed.